### ISSUE TITLE
Test unity 2019.1.0 alpha GUI Features

### ISSUE THREAD

**Issue by MystikalPooka** [MystikalPooka/Unity-Visual-Behavior-Tree#15]
_You might want to have a look at [this](https://www.youtube.com/watch?v=MNNURw0LeoQ) concerning the editor. If it's already available in the alpha version then you might want to start targeting it already._

_Originally posted by @ZiadJ in https://github.com/MystikalPooka/Unity-Visual-Behavior-Tree/issues/12#issuecomment-442008358_

**Comment by MystikalPooka** [comment]
https://github.com/Unity-Technologies/ShaderGraph/wiki/Custom-Nodes-With-CodeFunctionNode <-- this is a much better solution, I think. I'll get to this once I fix the streams on this major refactor.

**Comment by MystikalPooka** [comment]
Looking at it for about 8-9 hours and I'm still thinking this is the best solution. However, the SRP clone is 4.04 GB which is WAY too big to include in a package. Thankfully we don't need 99.9% of the stuff that comes with it including all f the shader nodes, shaders, and HLSL files. I stripped out just the blackboard editor stuff and it is down to 4 MB. I can strip out most of that as well once I finish using them as examples.

TL;DR Blackboard is a GO and should be less than a couple MB to implement!

**Comment by ZiadJ** [comment]
You might want to have a look at [this](https://github.com/lyback/Unity-behavior-tree) visual interface as well before making up your mind.
